src/app/static/src/app/components/Errors.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+    <div v-if="hasErrors" class="container">
+        <div class="content">
+            <div class="alert alert-danger alert-dismissible fade-show" role="alert">
+                <p>The following errors occurred. Please contact {{title}} support if this problem persists.</p>
```

Comment:
Duplicating instructions here? i.e. "Please contact... etc" will already be encompassed in the API error message? I think just remove this bit.

Reply:
The 'Please contact...' bit would be there if the error had originated in an exception in the backend, I guess there are other error cases that might need to be displayed in the front end though?

Reply:
Yeah, I've just made the front-end errors a bit more descriptive: https://github.com/mrc-ide/hint/pull/175/files#diff-0f0781cfcf5e78654409fce90c2c466fL55

They don't have the app name or email in there though - but we could get that in by having them as variables on the page rendered by freemarker?